getting-started/project-setup.adoc

```diff
@@ -9,7 +9,7 @@ Set up your Maven environment, following these instructions for doing so xref:..
 With the migration of the BAMOE product to Quarkus 3 you now have these options:
 
 * Create a project manually
-* Create a new project using the {CANVAS} accelerator functionality (see xref:../tools/applying-accelerators[Using Accelerators]).
+* Create a new project using the {CANVAS} accelerator functionality (see xref:../tools/applying-accelerators.adoc[Using Accelerators]).
```

Comment:
We've been doing .html instead of the adoc, I think due to IBM's documentation builds. I don't think it matters because it is moved to .html anyway during the build, but we might as well stay consistent.

Reply:
Hmm... ok, I'll change it.
My issue with having it as .html is that the preview doesn't work (at least on VSCode)

Reply:
I confirm that links to .adoc do work with the IBM build and I also appreciate the preview in VS Code so have been doing them this way.